app/src/main/java/com/github/orkest/ui/feed/PostViewModel.kt

```diff
@@ -94,8 +95,9 @@ open class PostViewModel {
     /**
      * Returns a list of the most recent posts since the last connection of the user from the database
      */
-    open fun getRecentPosts(lastConnected: LocalDateTime): CompletableFuture<List<Post>>{
-        return dbAPI.getRecentPostsFromDataBase(lastConnected)
+    open fun getRecentPosts(lastConnected: LocalDateTime,
+                            maxDaysNb: Long = DEFAULT_MAX_RECENT_DAYS ): CompletableFuture<List<Post>>{
+        return dbAPI.getRecentPostsFromDataBase(lastConnected, maxDaysNb)
     }
```

Comment:
I actually don't understand the purpose of giving maxDaysNb as a parameter to the function. It is not something that will be changed depending on the post so it shouldn't stay as a parameter. Only calling the constant MAX_RECENT_DAYS is enough and if we want to change this value, we would know exactly where to do so.

Reply:
There could be times where you want to load all posts that are 30 days before and others where you would like to load posts only for the last pas week at the maximum, it is therefore the most maintainable way to do it I think. The MAX_RECENT_DAYS value is actually the maximum allowed value since I am not handling the case for which one would like to load unseen posts more than 1 month old (Made the choice that it would become necessarily complexities with respect to performance and readability)

Reply:
What are the cases where we would want to load posts with different time limits ? Is it for caching purposes ?